Aquality.Selenium/src/Aquality.Selenium/Elements/Actions/JsActions.cs

```diff
@@ -159,17 +162,24 @@ public Point GetViewPortCoordinates()
 
         protected T ExecuteScript<T>(JavaScript scriptName, params object[] arguments)
         {
-            return Browser.ExecuteScript<T>(scriptName, element.GetElement(), arguments);
+            return Browser.ExecuteScript<T>(scriptName, ResolveArguments(arguments));
         }
 
         protected void ExecuteScript(JavaScript scriptName, params object[] arguments)
         {
-            Browser.ExecuteScript(scriptName, element.GetElement(), arguments);
+            Browser.ExecuteScript(scriptName, ResolveArguments(arguments));
         }
 
         protected internal void LogElementAction(string messageKey, params object[] args)
         {
             Logger.InfoLocElementAction(elementType, element.Name, messageKey, args);
         }
+
+        private object[] ResolveArguments(params object[] arguments)
```

Comment:
And what was the problem?

Reply:
problem is that ScrollBy method didn't work in previous version, because when we provide params like 
`Browser.ExecuteScript(scriptName, element, params);` 
method ExecuteScript see 3 parameters instead of dinamic number and take 'params' just as array, so script should be: 
`arguments[0].scrollBy(arguments[1][0], arguments[2][1]);`
Here, you can find fix )